demo/index.html

```diff
@@ -0,0 +1,372 @@
+            </svg>
+          </label>
+        </li>
+          <input class="toggle" type="checkbox" name="wave-btn" id="wave-btn" />
+          <label class="lbl" id="wave-lbl" for="wave-btn"><svg id="wave-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 53.5">
+            <path id="wave-path" d="M29.5,3.7c0.4,0,0.7,0.1,1.1,0.3c1.4,0.8,1.3,2.4,0.5,3.8c0,0-6.4,12.1-6.9,13.3c-0.3,0.8,0,1.2,0.4,1.2 c0.2,0,0.5-0.2,0.8-0.5C26.2,20.8,36,7.3,36,7.3c0.6-0.9,1.4-1.5,2.3-1.5c0.4,0,0.8,0.1,1.2,0.4c1.3,0.9,1.1,2.5,0.2,3.8 c0,0-9.6,14.2-10,14.8c-0.4,0.6-0.3,1.1,0.1,1.1c0.2,0,0.5-0.1,0.9-0.4c1.3-0.9,10.4-12,10.4-12c0.7-0.7,1.5-1.2,2.3-1.2 c0.6,0,1.1,0.2,1.6,0.6c1.2,1.1,0.6,2.6-0.5,3.8c0,0-9.6,12-10,12.5c-0.3,0.4-0.2,1.5,0.6,1.5c0.2,0,0.4-0.1,0.7-0.2l6.5-5.5 c0.7-0.7,1.8-1.1,2.7-1.1c0.6,0,1.2,0.2,1.6,0.5c0.7,0.4-0.1,2-0.3,2c-1,0.9-9,9.1-10.9,12C29.7,46.7,25,49.9,20.3,49.9 c-2.6,0-5.2-1-8-2.6C8,45,5.2,40.6,5,35.7C4.7,31.3,5.1,28.2,3.5,21c-0.6-2.9-1-5.7,0.8-5.7c0.3,0,0.8,0.1,1.3,0.3 c2.1,0.9,3.5,3,3.7,5.3c0.1,2.7,1.4,5.8,3,6.5c0.1,0,0.2,0.1,0.3,0.1c1.8,0,4.9-5.5,6.1-7.8c1.2-2.3,8.4-14.2,8.4-14.2 C27.7,4.4,28.6,3.7,29.5,3.7 M29.5,1.2L29.5,1.2c-1.8,0-3.5,1.1-4.6,2.9c-0.3,0.5-7.2,12-8.5,14.3c-1.7,3.2-3,5-3.7,5.9 c-0.4-0.8-0.9-2.1-1-3.6c-0.2-3.3-2.2-6.1-5.2-7.4C5.8,13,5,12.8,4.3,12.8c-1.1,0-2.1,0.5-2.8,1.3c-1.3,1.6-1.1,3.9-0.4,7.4 c1.1,5.1,1.2,8,1.3,11c0,1.1,0.1,2.2,0.1,3.3C2.7,41.5,6,46.8,11,49.5c3.5,2,6.4,2.9,9.3,2.9c5.8,0,11.2-3.9,17.2-12.6 c1.7-2.6,9-10.1,10.4-11.4c1-0.7,1.5-2.1,1.6-2.9c0.2-1.3-0.4-2.6-1.4-3.2c-0.9-0.6-1.9-0.9-3-0.9c-0.3,0-0.7,0-1,0.1 c1.2-1.5,2.2-2.7,2.5-3.1c1.2-1.4,1.8-2.8,1.8-4.2c-0.1-1.2-0.6-2.2-1.5-3c-0.9-0.8-2-1.3-3.2-1.3c-0.3,0-0.7,0-1,0.1 c0.3-0.7,0.4-1.3,0.4-2c0-1.5-0.7-2.9-2-3.8c-0.8-0.6-1.7-0.8-2.6-0.8c-1.6,0-3,0.8-4.2,2.3c0,0,0,0,0,0c-0.1-1.5-0.9-2.8-2.3-3.6 C31.1,1.5,30.3,1.2,29.5,1.2L29.5,1.2z"/>
+        </svg></label>
+      </ul>
+    </div>
+  </div>
+
+  <div id="content">
```

Comment:
Since it's such a small tool I figured it'd be best not to overdo it by having too many pages, and instead just link to the relevant info. What do you think?

Reply:
I agree that the page should have minimal information.

One thing that comes to mind is that any changes to the README would require changes it both places - but I think this is already 100x better than what we have now so I think we can keep it and just iterate as we go if we want to.